middleware/requestid/requestid.go

```diff
@@ -36,6 +38,10 @@ func New(config ...Config) fiber.Handler {
 		// Add the request ID to locals
 		c.Locals(requestIDKey, rid)
 
+		// Add the request ID to UserContext
+		ctx := context.WithValue(c.UserContext(), requestIDKey, rid)
```

Comment:
After https://github.com/gofiber/fiber/pull/3193 is merged this will be `c.Context()`

Reply:
> After #3193 is merged this will be `c.Context()`

If I change it now, I will get an error locally